.circleci/config.yml

```diff
@@ -311,42 +339,88 @@ jobs:
 
   test-backend:
     executor: machine-executor
-    parameters:
-
-      owasp:
-        type: boolean
-        default: true
-
-      qasp:
-        type: boolean
-        default: true
-
-    branches:
     steps:
       - checkout
       - docker-compose-check
+      - docker-compose-up-backend
       - run:
-          name: "Test: Build and Spin-up Django API Service"
```

Comment:
These quotes were only needed because of the colon, but this is just minor cleanup.